### PR TITLE
fix $this errors

### PR DESCRIPTION
**Pull request by The-Voidwalker**

### CODE REVIEW

File: RemoteWiki.php [hunk 1]
```diff
@@ -3,6 +3,7 @@ class RemoteWiki {
 	protected $db;
```
Comment (Southparkfan) [inline RemoteWiki.php]: Remove this

File: RemoteWiki.php [hunk 2]
```diff
@@ -3,6 +3,7 @@ class RemoteWiki {
...
 
 	private function __construct( $dbname, $sitename, $language, $private, $closed, $closedDate, $inactive, $inactiveDate, $settings ) {
+		$this->db = wfGetDB( DB_MASTER, [], 'metawiki' );
```
Comment (Southparkfan) [inline RemoteWiki.php]: Remove this

Comment (Southparkfan) [reply RemoteWiki.php]: And change line 47 to "$res = wfGetDB( DB_MASTER, [], 'metawiki' )->selectField("

**Comment by Reception123** [comment]
Seems fine to me, but I will wait for someone more experienced in this domain (@JohnFLewis @Southparkfan) to approve first.

**Comment by Reception123** [comment]
@JohnFLewis @Southparkfan Could you please review so that we can get ManageWiki operational again?

**Comment by JohnFLewis** [comment]
It looks okay to me

**Pull request merged**